README_SKINNY.rst

```diff
@@ -0,0 +1,14 @@
+=============================================
+MLflow Skinny: A Lightweight Machine Learning Lifecycle Platform Client
+=============================================
+
+MLflow Skinny is a lightweight MLflow package without SQL storage, server, UI, or data science dependencies.
+MLflow Skinny supports:
+* Tracking operations (logging / loading / searching params, metrics, tags + logging / loading artifacts
+* Model registration, search, artifact loading, and transitions
+* Execution of GitHub projects within notebook & against a remote target.
```

Comment:
For some reason, `twine` complains about the RST formatting when uploading to test PyPI. I think that the spacing before / after list items may be the culprit:

```suggestion
MLflow Skinny is a lightweight MLflow package without SQL storage, server, UI, or data science dependencies.

MLflow Skinny supports:

* Tracking operations (logging / loading / searching params, metrics, tags + logging / loading artifacts
* Model registration, search, artifact loading, and transitions
* Execution of GitHub projects within notebook & against a remote target.
```